docs/_static/custom.js

```diff
@@ -0,0 +1,16 @@
+document.addEventListener("DOMContentLoaded", function () {
+  var script = document.createElement("script");
+  script.type = "module";
+  script.id = "runllm-widget-script"
+
+  script.src = "https://cdn.jsdelivr.net/npm/@runllm/search-widget@stable/dist/run-llm-search-widget.es.js";
+
+  script.setAttribute("version", "stable");
+  script.setAttribute("runllm-keyboard-shortcut", "Mod+j"); // cmd-j or ctrl-j to open the widget.
```

Comment:
I opened the generated docs for this PR but wasn't able to open the runllm assistant neither with cmd-j nor ctrl-j. How is this supposed to work?

Reply:
@YarShev there's a button on the bottom right of the docs to open the prompt.

@vsreekanti this is a good question, is it possible to test this? When I tried this is the error that I got: 
<img width="1113" alt="Screenshot 2024-06-25 at 9 35 37 AM" src="https://github.com/modin-project/modin/assets/149107610/675975a5-188c-4068-bf94-e24848f88292">

Reply:
@sfc-gh-dpetersohn, yes absolutely! We use Recaptcha to prevent spam requests, so we just need to add the URL of the site to the allowed list. Can you share the URL you're testing from?

Reply:
https://modin--7326.org.readthedocs.build/en/7326/

This is the PR docs build

Reply:
> @YarShev there's a button on the bottom right of the docs to open the prompt.

Hm... I don't see any button.

Reply:
Ask AI button in the built docs

<img width="1341" alt="Screenshot 2024-06-25 at 1 32 35 PM" src="https://github.com/modin-project/modin/assets/149107610/b5ff6d84-6f14-42ea-a2f6-dccbc6df85f2">

Reply:
Can you try testing again? Just added both the test domain and the prod domain to our Recaptcha instance!